CMakeLists.txt

```diff
@@ -172,6 +172,7 @@ add_subdirectory(src/CaptureFile)
 add_subdirectory(src/CaptureServiceBase)
 add_subdirectory(src/CaptureUploader)
 add_subdirectory(src/ClientData)
+add_subdirectory(src/ClientFlags)
```

Comment:
Yeah, I know `OrbitApp` is not available here, but that was kind of my point, because using `OrbitApp::IsDevMode` "restricts" how far down we were using this flag. If you look at what modules depend on `ClientFlags`, I think it's only actual UI modules, because usually these flags simply control whether a piece of the UI should be shown or not.

But if we just use `ClientFlags` anywhere we need access to some flag, I'm worried things can get out of hand, for example by accidentally using flags in modules that could be shared by multiple components or binaries, some of which are not supposed to receive those flags in the first place.

Reply:
For consistency:discussed the options offline with Florian and agreed to keep it as is.